### DISCUSSION TITLE
Getting HTML instead of JSON while using HTTP GET

### DISCUSSION THREAD

**Discussion by BobSquarePants** [osm-search/Nominatim#3033] [Q&A] [answered]
Hi everyone,

I was making some request from frontend to https://nominatim.openstreetmap.org/search
Now I'm trying on the server side with [request](https://requests.readthedocs.io/en/latest/)

but instead of receiving the data back as json, I'm receiving an html page

extract:
```html
 <title>Nominatim Demo</title>
```
I realized that with `requests` there is no referee nor user-agent send by default, I have fixed that.

Is that why I'm receiving a webpage ? How long is the exclusion ?

Thanks.

**Comment by lonvia** [comment, marked as answer]
If you want to receive json you have to add `format=json` (or jsonv2 or geojson or geocodejson) to the URL.

**Reply by BobSquarePants** [reply]
oh ! damn I forgot that on the server side Thanks !!!
